packages/terra-markdown/src/Markdown.jsx

```diff
@@ -22,12 +25,16 @@ const defaultProps = {
   src: '',
 };
 
+const markdownClasses = cx([
+  'markdown-body',
+]);
+
 
 /* eslint react/no-danger:0 */
 const Markdown = props => (
   <div
     dir="ltr"
-    className="markdown-body"
+    className={markdownClasses}
```

Comment:
We'll want to keep this class name global as-is, `markdown-body`, instead of hashing it with `classNames.bind(styles);`.

This package [pulls in a third-party-library](https://github.com/cerner/terra-core/blob/master/packages/terra-markdown/package.json#L28) called [github-markdown-css](https://github.com/sindresorhus/github-markdown-css/tree/v2.10.0) which uses [the `markdown-body` class to scope the CSS styles](https://github.com/sindresorhus/github-markdown-css/blob/v2.10.0/github-markdown.css#L6). If we hash the class name by using  `classNames.bind(styles);`, the third-party styles from github-markdown-css won't apply as intended.

This is likely why the wdio tests for some of the other packages that pull un terra-markdown failed on TravisCI. If you update this class back to how it was before, it should pass.